app/views/groups/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<% if user_signed_in? %>
+  <nav class="blue-navigation">
+  <%= link_to '←', root_path, class: 'nav-buttons first'%>
+  <h1>Groups</h1>
+  <%= button_to 'Logout', destroy_user_session_path, method: :delete, class: 'nav-buttons second' %>
+  <% else %>
+  <%= button_to 'Login', new_user_session_path, class: 'nav-buttons second display-none' %>
+  <% end %>
+</nav>
+
+<br/>
+<p style="color: green"><%= notice %></p>
+<br/>
+
+<div class="main-container">
+  <div id="groups">
+    <% @groups.each do |group| %>
+      <%= link_to group_path(group), class: "group-link" do %>
+        <%= render group %>
+      <% end %>
+    <% end %>
+  </div>
+  
+  <div class="footer-buttons">
+  <%= link_to "View Clothes", group_clothes_path(:group_id), class:"inner-buttons"%>

```

Comment:
- [ ] I really don't know what you mean by view clothes, do you mean view transactions?